s3/replication/common/src/s3replicationcommon/jobs.py

```diff
@@ -320,6 +330,8 @@ def _remove_job(self, replication_id):
                 self._jobs_paused.remove(replication_id)
             else:
                 # COMPLETED/ABORTED/FAILED.
+                self._logger.debug("Jobs[{}]: Removed job with job_id {}.".
```

Comment:
You may remove this debug log to maintain the consistency with other cases. We are not printing anything on console while removal of any job.

To optimize above code, 
You may make the get_state() call just once. carry the return in the variable and then use that variable in if else ladder.
e.g.
state = job.get_state()
if state == JobState.INITIAL:
...
elif state == JobState.RUNNINING:
...
...

Reply:
done